base/mem_trie.hpp

```diff
@@ -59,6 +59,16 @@ class MemTrie
       ForEachInNode(*root, prefix, std::forward<ToDo>(toDo));
   }
 
+  // Calls |toDo| for each value in the node that is reachable
+  // by |prefix| from the trie root. Does nothing if such node does
+  // not exist.
+  template <typename ToDo>
+  void ForEachValueInNode(String const & prefix, ToDo && toDo) const
```

Comment:
In order to have a way to remove the unnecessary argument in compile-time.
As per offline discussion, we found another way to do it.